platform/ios/src/MGLAnnotationContainerView.h

```diff
@@ -0,0 +1,17 @@
+#import <UIKit/UIKit.h>
+
+#import "MGLTypes.h"
+
+@class MGLAnnotationView;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MGLAnnotationContainerView : UIView
+
++ (instancetype)annotationContainerViewWithAnnotationContainerView:(nonnull MGLAnnotationContainerView *)annotationContainerView;
```

Comment:
`nonnull` is unnecessary here because of the `NS_ASSUME_NONNULL_BEGIN` guard above.